io/disk/Avago_storage_adapter/avago3008.py

```diff
@@ -44,6 +44,7 @@ def setUp(self):
         self.disk = str(self.params.get('disk')).split(" ")
         self.spare = str(self.params.get('spare'))
         self.size = int(self.params.get('size', default='max'))
+        self.tool_location = self.params.get('tool_location')
```

Comment:
@vrbagalkote I would suggest to skip the test in case the tool is not present.

```if not os.path.isfile(self.tool_location) or not os.access(self.tool_location, os.X_OK):
    self.skip()```